x-pack/plugins/siem/public/common/mock/global_state.ts

```diff
@@ -27,11 +27,10 @@ import { networkModel } from '../../network/store';
 import { TimelineType, TimelineStatus } from '../../../common/types/timeline';
 import { initialAlertListState } from '../../endpoint_alerts/store/reducer';
 import { initialHostListState } from '../../endpoint_hosts/store/reducer';
-import { getManagementInitialState } from '../../management/store';
-
-const alertList = initialAlertListState();
-const hostList = initialHostListState();
-const management = getManagementInitialState();
+import { initialManagementState } from '../../management/store/reducer';
+import { AlertListState } from '../../../common/endpoint_alerts/types';
+import { HostState } from '../../endpoint_hosts/types';
+import { ManagementState } from '../../management/types';
```

Comment:
@paul-tavares i hoisted the type for management state up, but after our convo yesterday, I wanted to check w/ you on this.

Reply:
That's cool with me. What was the primary reason for the move? (I assume simpler import paths ++ avoid the circular dep. check)

Reply:
yea i moved it because i was trying to get around some circluar deps stuff but it could probably be moved back w/o issue if you'd prefer it in the original place

Reply:
if all sub-plugins keep this within their own directly structure, then I would say lets move it back. But if there is no pattern/consistency there, then I'm also ok with it being hoisted here.

Reply:
it's still in the subplugin directory structure.